sig-k8s-infra/annual-report-2021.md

```diff
@@ -39,44 +35,43 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - TBD
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - TBD
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - TBD
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - TBD
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - TBD
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - AWS
+   - Google
+   - II Coop
+   - Independent
+   - VMware
```

Comment:
Are these actually the answers to question 5?

Reply:
No. I just opened the PR. I'll fix the report with the correct answers before final review. Apologies for the mistakes introduced. :-)